examples/spin-timer/trigger-timer.json

```diff
@@ -0,0 +1,16 @@
+{
+    "name": "trigger-timer",
+    "description": "IT IS I THE GREAT OZ",
+    "homepage": "www.example.com",
+    "version": "0.1.0",
+    "spinCompatibility": ">=0.7",
+    "license": "Apache-2.0",
+    "packages": [
+        {
+            "os": "linux",
```

Comment:
Would it be possible to defer that to a later PR?  I'm concerned about how we manage and test cross-compilations.  Although completely recognise that a lot of folks are on Macs so the Linux setting is not useful to them.

What I would really like to do is create a `spin-pluginify` plugin that takes a (Rust) project, cross-compiles it, tars all the various outputs, generates hashes, and updates the manifest instead of muggins here having to do it by hand...!

Reply:
sounds good! Also love that idea about `spin-pluginify`